tests/database-manager.spec.js

```diff
@@ -232,7 +232,21 @@ _.map(availableDatabases, function(db) {
 
     it('#populateDb should populate data from given directory', function() {
       return dbManager
-        .populateDb(__dirname + '/populate/*.js')
+        .populateDb(__dirname + '/populate/*-data.js')
```

Comment:
Was these *-data.js files converted to new format or am I blind? 😅

Reply:
Hi Mikael,

Just got back from vacations, here's the explanation:

```js
// $ cat tests/populate/01-user-data.js
module.exports = function(knex) {
  return new Promise(function(resolve) {
    setTimeout(function() {
      return knex('User')
        .insert({
          username: 'dummy',
          email: 'lol@fake.invalid',
        })
        .then(resolve);
    }, 100);
  });
};

// $ cat tests/populate/sample-data-old.js
exports.seed = function(knex) {
  return knex('User').insert({
    username: 'dummy',
    email: 'lol@fake.invalid',
  });
};
```

AFAIS `-data.js` files are already in the new seed format, whereas this PR adds a `sample-data-old.js` for the next test to pass.

Do you see an issue with this?

Cheers

Reply:
Ah right. Yep all good here then.

Reply:
Thanks.
Cheers